examples/tutorials/self-paced-training/part-4_advanced_federated_learning/chapter-7_algorithms_and_workflows/07.2_algorithms/07.2.2_cyclic/cyclic_weight_transfer_example.ipynb

```diff
@@ -1,19 +1,308 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "514c47e2-420d-4af4-9bf0-cac337c51c39",
+   "metadata": {},
+   "source": [
+    "# Cyclic Weight Transfer (CWT) with Cyclic Workflow\n",
+    "\n",
+    "In this example, we will demonstrate the Cyclic workflow using the Client API with the CIFAR10 dataset. \n",
+    "\n",
+    "## Cyclic Workflow\n",
+    "<a id = \"cyclic_workflow\"></a>\n",
+    "\n",
+    "[Cyclic Weight Transfer](https://pubmed.ncbi.nlm.nih.gov/29617797) (CWT) uses the server-controlled `CyclicController` to pass the model weights from one site to the next in a cyclic fashion. \n",
+    "\n",
+    "In the Cyclic workflow, sites train one at a time, while sending the model to the next site. The order of the sites can be specified as fixed, random, or random (without same in a row).  A round is finished once all sites in the defined order have completed training once, and the final result is returned to the server. This differs from Scatter-and-Gather or FedAvg workflows, wherein all sites train simultaneously and aggregate their results together at the end of a round.\n",
```

Comment:
a round is finished once all sites (or a pre-defined number of sites considering the random case?)

Reply:
The description seems correct. Even in the random order, each client needs to complete their training.